### ISSUE TITLE
Expose interface returned by createClient

### ISSUE THREAD

**Issue by Bullrich** [polkadot-api/polkadot-api#324]
I don't know if it is available yet, but after experimenting I was unable to access the type for the createClient method.

It would be nice if the type returned by `createClient` could be accessed for storing it.

<details><summary>Example:</summary>

```ts
// Client is currently not exported
import { createClient, Client } from "@polkadot-api/client";
import { getChain } from "@polkadot-api/node-polkadot-provider";
import { WebSocketProvider } from "@polkadot-api/ws-provider/node";

// We currently can not do this
let client: Client;


beforeEach(() => {
  client = createClient(
    getChain({
      provider: WebSocketProvider("wss://rpc.polkadot.io"),
      keyring: [],
    }),
  );
});

test(("A") => {
 // we test something
});

afterEach(() => {
  client.destroy()
});
```
</details>

If it is already available, I apologize for the unnecessary request.

**Comment by josepot** [comment]
Yeah! We are aware of this issue, and we are currently working on this. Thanks for raising it, though. So, yeah, there are a bunch of public interfaces that are not being exported, then there are others that are being exported that they shouldn't be exported, and then there are others that are exported with names that are not the "best" ones.

We are hopping to have addressed in the next couple of days. So, I'm leaving this issue opened.

In the meanwhile you can do this, though:

```ts
let client: ReturnType<typeof createClient>
```

**Comment by Bullrich** [comment]
> In the meanwhile you can do this, though:
> 
> ```ts
> let client: ReturnType<typeof createClient>
> ```

I can use this as a workaround 😄. Thanks!

**Comment by josepot** [comment]
Closing this, as this was fixed on #325
